.github/workflows/docker.yml

```diff
@@ -0,0 +1,58 @@
+name: Create and publish Docker images
+
+on:
+  push: { branches: [ unstable ] }
+  pull_request: { branches: [ unstable ] }
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - name: github_ci
+        #- name: jenkins_ci # flatironinstitute/cthyb:master-ubuntu-clang not accessible
+        - name: mpich
+        - name: openmpi
+        #- name: triqs_default # flatironinstitute/triqs:unstable-ubuntu-clang not accessible
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Docker buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Log in to the Container registry
+        if: github.event_name != 'pull_request'
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ghcr.io/${{ github.repository }}_${{ matrix.name }}
```

Comment:
The final docker images land on ghcr.io I assume? So we should then remove the once on docker hub?

Reply:
You can keep both. I don't see any harm.